examples/material-ui-remix-ts/package.json

```diff
@@ -4,16 +4,17 @@
   "private": true,
   "scripts": {
     "build": "remix build",
-    "dev": "remix dev",
-    "postinstall": "remix setup node",
```

Comment:
this line can be removed on v2.

```console
$ npx remix setup node
WARNING: The setup command is no longer necessary as of v2. This is a no-op. Please remove this from your dev and CI scripts, as it will be removed in v3.
```